### ISSUE TITLE
LaunchdarklyFlutterPlugin.java issue on line 63

### ISSUE THREAD

**Issue by stanojevicboban** [andre-paraense/launchdarkly_flutter#28]
**Describe the bug**
The issue doesn't appear on every device, but currently I see it is occuring on Samsung Galaxy devices - if that has any significance.

**To Reproduce**
This happens on app load when the class is being initialized.

**Expected behavior**
Launch darkly 

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy A71 5G
 - OS: Android 11

**Additional context**
Stack:

```
Fatal Exception: java.lang.RuntimeException
Unable to destroy activity {io.superphone.app/io.superphone.app.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.launchdarkly.android.LDClient.close()' on a null object reference
```

```
com.oakam.launchdarkly_flutter.LaunchdarklyFlutterPlugin.onDetachedFromEngine (LaunchdarklyFlutterPlugin.java:63)
io.flutter.embedding.engine.FlutterEngineConnectionRegistry.remove (FlutterEngineConnectionRegistry.java:262)
io.flutter.embedding.engine.FlutterEngineConnectionRegistry.remove (FlutterEngineConnectionRegistry.java:270)
io.flutter.embedding.engine.FlutterEngineConnectionRegistry.removeAll (FlutterEngineConnectionRegistry.java:278)
io.flutter.embedding.engine.FlutterEngineConnectionRegistry.destroy (FlutterEngineConnectionRegistry.java:122)
io.flutter.embedding.engine.FlutterEngine.destroy (FlutterEngine.java:373)
io.flutter.embedding.android.FlutterActivityAndFragmentDelegate.onDetach (FlutterActivityAndFragmentDelegate.java:555)
io.flutter.embedding.android.FlutterActivity.release (FlutterActivity.java:589)
io.flutter.embedding.android.FlutterActivity.onDestroy (FlutterActivity.java:610)
android.app.Activity.performDestroy (Activity.java:8444)
android.app.Instrumentation.callActivityOnDestroy (Instrumentation.java:1344)
android.app.ActivityThread.performDestroyActivity (ActivityThread.java:5556)
android.app.ActivityThread.handleDestroyActivity (ActivityThread.java:5610)
android.app.servertransaction.DestroyActivityItem.execute (DestroyActivityItem.java:44)
android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:176)
android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:97)
android.app.ActivityThread$H.handleMessage (ActivityThread.java:2295)
android.os.Handler.dispatchMessage (Handler.java:106)
android.os.Looper.loop (Looper.java:246)
android.app.ActivityThread.main (ActivityThread.java:8443)
java.lang.reflect.Method.invoke (Method.java)
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:596)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1130)
```

**Comment by andre-paraense** [comment]
Thanks, @stanojevicboban , we will be looking into that soon.

**Comment by andre-paraense** [comment]
Fixed by #27 . New version will follow soon.
